lib/readGltfGeometry.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
```

Comment:
The words `read` and `write` for these two files are slightly confusing. Maybe `convert` is better.